mor1kx.core

```diff
@@ -87,3 +89,12 @@ targets:
       - "tool_rivierapro? (monitor)"
       - "tool_xsim? (monitor)"
     parameters: [trace_enable, trace_to_screen]
+
+  tinyfpga_bx:
```

Comment:
If we're only using this target to get the synthesis stats, then we never run the pnr tool and can simplify this a bit by dropping `nextpnr_options` and setting `pnr : none` directly in the core file. This will also allow us to get rid of the pcf file and the `tinyfpga_bx` fileset. When we do that we should also rename the target to something more descriptive, such as `synth` or something like that

There is even some further improvements to be done, but before that I would like to know if it is a mistake, or by design that the fpu fileset is not included

Reply:
Thank you @olofk, I will do the changes. Other than this fpu was not included intentionally, because with it that it gave the error: https://paste.ubuntu.com/p/VB2CPkckhv/

Reply:
Thanks @olofk for the comments this should have had a TODO comment,  disabling FPU was just a temporary measure to get the design to synthesize without requiring investigation into the verilog fixes.  It looks like some verilator annotations/comments are causing the failure.  @bandvig Do you think we can just remove them?

Reply:
We don't need to remove them. It's sufficient to just move it to after the case statement (`case (generic_cmp_opc_i) // synthesis parallel_case`). Seems like yosys is a bit stricter about that than many other tools.

With that said, if we include the fpu when doing the synthesis stats, then we can do further simplifications of the core file, but perhaps we should make the changes that we already talked about first

Reply:
@olofk @stffrdhrn 
I should note that 'synthesis parallel_case' is used in MAROCCHINO many times. If moving such meta-comments just after case statement (as Olof proposed) desn't hurt behavior of other tools, I agree.

Reply:
I confirmed that @olofk 's fix for parallel case allows the build to work now.  @Nancy-Chauhan  you can add back fpu.